test/cql-pytest/cassandra_tests/validation/entities/json_test.py

```diff
@@ -133,8 +133,8 @@ def testSelectJsonWithPagingWithFrozenUDT(cql, test_keyspace):
                           ["{\"a\": 1, \"b\": 2, \"c\": [\"1\", \"2\"]}", "[\"" + str(uuid) + "\", 3]", "3"],
                           ["{\"a\": 1, \"b\": 2, \"c\": [\"1\", \"2\"]}", "[\"" + str(uuid) + "\", 4]", "4"])
 
-# Reproduces issue #7911, #7912, #7914, #7915, #7944, #7954
-@pytest.mark.xfail(reason="issues #7912, #7914, #7915, #7944, #7954")
+# Reproduces issue #7911, #7912, #7914, #7944, #7954
+@pytest.mark.xfail(reason="issues #7912, #7914, #7944, #7954")
```

Comment:
After fixing 7915, we can remove it from the xfail list, but not from the comment above - I believe the comment is still useful to understand why this is a useful test and which bugs it caught in the past - even after these issues have been fixed.